src/lib.rs

```diff
@@ -8,35 +8,270 @@ mod transaction;
 pub mod update;
 
 use crate::{
-    keys::{OwnershipKeyPair, OwnershipPublicKey, PublishingKeyPair, RevocationKeyPair},
+    create::{BuildFundingPSBT, SignFundingPSBT},
+    keys::{
+        OwnershipKeyPair, OwnershipPublicKey, PublishingKeyPair, PublishingPublicKey,
+        RevocationKeyPair, RevocationPublicKey, RevocationSecretKey,
+    },
     transaction::{CommitTransaction, FundingTransaction, SplitTransaction},
 };
 use anyhow::bail;
-use bitcoin::Amount;
+use bitcoin::{Amount, Transaction};
 use ecdsa_fun::adaptor::EncryptedSignature;
-use keys::{PublishingPublicKey, RevocationPublicKey, RevocationSecretKey};
+
+#[allow(dead_code)]
```

Comment:
Pretty sure this is not needed as it is `pub` in the `lib.rs`

Reply:
It was indeed needed but now we are accessing TX_f from the outside. On my machine I get no clippy complaints.